src/components/FilterComponent/index.js

```diff
@@ -0,0 +1,74 @@
+import { useMainContext } from "../../context/main-context";
```

Comment:
export `main-context.js` from `index.js` in context folder, so it will become
```suggestion
import { useMainContext } from "../../context";
```

Reply:
done